ceph-releases/ALL/debian/stretch-backports/__OS_CODENAME__

```diff
@@ -0,0 +1 @@
+stretch
```

Comment:
__OS_CODENAME__ is then defined but never used.

Reply:
stretch-backports repo is needed for installing nfs-ganesha. So add this repo again, so that OS_CODENAME will be used.